services/simcore/.env

```diff
@@ -10,49 +10,53 @@ CATALOG_HOST=${PREFIX_STACK_NAME}_catalog
 
 DIRECTOR_HOST=${PREFIX_STACK_NAME}_director
 
-DOCKER_IMAGE_TAG=staging-latest
+DOCKER_IMAGE_TAG=master-latest
 
 EXTRA_HOSTS_SUFFIX=osparc.local
 
 OPS_MONITORED_NETWORK=monitored_network
 OPS_PUBLIC_NETWORK=public-network
 
 POSTGRES_DB=simcoredb
-POSTGRES_ENDPOINT=${PREFIX_STACK_NAME}_postgres:5432
-POSTGRES_HOST=${PREFIX_STACK_NAME}_postgres
-
+POSTGRES_ENDPOINT=simcore_postgres:5432
+POSTGRES_HOST=simcore_postgres
 POSTGRES_PASSWORD=adminadmin
 POSTGRES_PORT=5432
 POSTGRES_USER=scu
 
-PUBLISHED_HOST_NAME=staging.osparc.local
+PUBLISHED_HOST_NAME=osparc.local
```

Comment:
once all the things are ready. we need to use jinja2 or envsusbt instead of sed.

This way we could have things like:
PUBLISHED_HOST_NAME={{ repo.config.host_name }}